extension/chrome/settings/index.ts

```diff
@@ -526,19 +526,22 @@ View.run(
         $('.action_remove_key').on(
           'click',
           this.setHandler(async target => {
-            // the UI below only gets rendered when account_email is available
-            const family = $(target).data('type') as string;
-            const id = $(target).data('id') as string;
-            const longid = $(target).data('longid') as string;
-            if (family === 'openpgp' || family === 'x509') {
-              /* eslint-disable @typescript-eslint/no-non-null-assertion */
-              await KeyStore.remove(this.acctEmail!, { family, id });
-              await PassphraseStore.set('local', this.acctEmail!, { longid }, undefined);
-              await PassphraseStore.set('session', this.acctEmail!, { longid }, undefined);
-              /* eslint-enable @typescript-eslint/no-non-null-assertion */
-              this.reload(true);
-            } else {
-              Catch.report(`unexpected key family: ${family}`);
+            const fingerprint = $(target).data('fingerprint') as string;
+            if (await Ui.modal.confirm(`Are you sure you want to remove encryption key with fingerprint ${Str.spaced(Xss.escape(fingerprint))}`)) {
```

Comment:
let's move modal text string to `lang.ts` and also add question mark `?` in the end

Reply:
Sounds good!